### PR TITLE
doc: Fix Broken Links

### PR DESCRIPTION
**Pull request by justindhillon**
### Summery

Here is what I have fixed:

http://voorloopnul.com/blog/a-python-netstat-in-less-than-100-lines-of-code/
 --> https://web.archive.org/web/20190424172231/http://voorloopnul.com/blog/a-python-netstat-in-less-than-100-lines-of-code/

### Support my work

These links were found with [link-inspector](https://github.com/justindhillon/link-inspector). If you find this PR useful, give the repo a ⭐

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Translations
 ------------
 
 Changes to translations as well as new translations can be submitted to
-[Bitcoin Core's Transifex page](https://www.transifex.com/bitcoin/bitcoin/).
+[Bitcoin Core's Transifex page](https://explore.transifex.com/bitcoin/bitcoin/).
```
Comment (hebasto) [inline README.md:72]: The current link works fine for me. It redirects to https://app.transifex.com/bitcoin/bitcoin/dashboard/ or https://explore.transifex.com/bitcoin/bitcoin/ depending on website cookies content.

**Comment by hebasto** [comment]
@justindhillon 

The lint CI task fails because you modified the code in [subtrees](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#subtrees).

**Comment by fjahr** [comment]
And could you squash these to one commit, please?

**Comment by justindhillon** [comment]
The sub-trees are now unmodified, and everything is squashed into one commit. Thank you for all the help!

File: README.md (hunk 1 above)
Comment (fjahr) [reply README.md:72]: It works for me as well, so not absolutely needed to update it.

**Comment by fjahr** [comment]
ACK c9bf07cc64d10a33754c5f292a13d7d587b37757

The transifex links are not broken for me, so I'm not sure if they need updating. Maybe some browsers have issues with the redirect so it's ok for me to keep the change though.

File: README.md (hunk 1 above)
Comment (brunoerg) [reply README.md:72]: It works here as well.

Comment (hebasto) [reply README.md:72]: If a translator has already logged into their Transifex account, the current link brings them directly to the project dashboard. But the suggested link brings to the project in "explore" mode, and logging in is required to start working on translations.

So I'm NACK about this change.

**Comment by fanquake** [comment]
Given the above discussion, I think the only change that can be made here is the one in `netutil.py`. So you can either reduce this PR to just that change, or close it.

**Comment by justindhillon** [comment]
> Given the above discussion, I think the only change that can be made here is the one in `netutil.py`. So you can either reduce this PR to just that change, or close it.

Thanks for the feedback. The PR has been updated to only include ```netutil.pu```.

**Comment by fjahr** [comment]
ACK 6fa61e35320ac2bc623a9c9ca11b270b34e2d05a

**Pull request merged**